session/session.go

```diff
@@ -127,16 +127,6 @@ type Session interface {
 	ExecuteStmt(context.Context, ast.StmtNode) (sqlexec.RecordSet, error)
 	// Parse is deprecated, use ParseWithParams() instead.
 	Parse(ctx context.Context, sql string) ([]ast.StmtNode, error)
```

Comment:
If we remove `ParseWithParams`, this comment is now incorrect.

Btw: I discovered in one of my PRs that `Parse` can not be deprecated. It is important to have a Parse function which does not use placeholders so that a statement like this one can be parsed and then executed by `ExecuteStmt`:

```
mysql> select count(*) from t1 where formatted like '%ndan%';                                                                                                                                     |
ERROR 1105 (HY000): missing arguments, need 1-th arg, but only got 0 args 
```

Reply:
okay, when I finish the rewrite, I'll clean up some deprecated methods and update the comment here.

Reply:
@morgo can you give me link on that `Parse`?